pyproject.toml

```diff
@@ -1,5 +1,72 @@
 [build-system]
-requires = ["setuptools>=45", "wheel", "setuptools_scm>=6.2<8"]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "linopy"
+version = "0.3.9"
+description = "Linear optimization with N-D labeled arrays in Python"
+readme = "README.md"
+authors = [{ name = "Fabian Hofmann", email = "fabianmarikhofmann@gmail.com" }]
+license = { file = "LICENSE" }
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Environment :: Console",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: MIT License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+]
+
+
+dependencies = [
+    "numpy",
+    "scipy",
+    "bottleneck",
+    "toolz",
+    "numexpr",
+    "xarray>=2023.9.0",
+    "dask>=0.18.0",
+    "tqdm",
+    "deprecation",
+]
+
+[project.urls]
+Homepage = "https://github.com/PyPSA/linopy"
+Source = "https://github.com/PyPSA/linopy"
+
+[project.optional-dependencies]
+docs = [
+    "ipython",
+    "numpydoc",
+    "sphinx",
+    "sphinx_rtd_theme",
+    "sphinx_book_theme",
+    "nbsphinx",
+    "nbsphinx-link",
+    "gurobipy",
+    "ipykernel",
+    "matplotlib",
+]
+dev = [
+    "pytest",
+    "pytest-cov",
+    "netcdf4",
+    "pre-commit",
+    "paramiko",
+    "gurobipy",
+    "highspy",
+]
+solvers = [
+    "gurobipy",
+    "highspy",
+    "cplex",
+    "mosek",
+    "mindoptpy",
+    "coptpy",
+    "xpress; platform_system != 'Darwin' and python_version < '3.11'",
+    "pyscipopt; platform_system != 'Darwin'",
+]
 
 [tool.setuptools_scm]
 write_to = "linopy/version.py"
```

Comment:
You can remove this line as well as the `version.py` file as setuptools-scm handles setting the correct version internally.

To also provide version information in the conventional `linopy.__version__` you then only need to add my snippet from the comment before :)

Reply:
forgot that one, thanks